libs/designsystem/item/src/item.component.spec.ts

```diff
@@ -92,6 +95,66 @@ describe('ItemComponent', () => {
         '--inner-padding-end': size('xxs'),
       });
     });
+
+    it('should make the disclosure wrapper element get the same height as its child (kirby-icon)', async () => {
+      spectator.setInput('disclosure', 'arrow-down');
+      spectator.detectChanges();
+
+      const disclosureWrapper = spectator.query<HTMLElement>('.disclosure');
+      const icon = spectator.query<HTMLElement>('kirby-icon');
+
+      expect(getComputedStyle(disclosureWrapper).height).toBe(getComputedStyle(icon).height);
```

Comment:
Consider simplifying and using existing test utils:
```suggestion
      expect(disclosureWrapper).toHaveComputedStyle(
        {
          height: getComputedStyle(icon).height),
        }
      );
```